### PR TITLE
Streamlined specs

### PR DESCRIPTION
**Pull request by atz**
Simple improvements

### CODE REVIEW

File: spec/actors/hyrax/actors/generic_work_actor_spec.rb [hunk 1]
```diff
@@ -242,6 +236,7 @@
         expect(old_parent.reload.members).to eq []
       end
     end
+
```
Comment (jcoyne) [inline spec/actors/hyrax/actors/generic_work_actor_spec.rb]: I think the breaks between context blocks leads to readability.  Do you feel strongly that  we need to compress everything?

File: spec/actors/hyrax/actors/generic_work_actor_spec.rb [hunk 2]
```diff
@@ -204,11 +201,8 @@
   describe '#update' do
     let(:curation_concern) { create(:generic_work, user: user, admin_set_id: admin_set.id) }
 
-    subject { Hyrax::CurationConcern.actor }
-
     context 'failure' do
       let(:attributes) { {} }
-
```
Comment (jcoyne) [inline spec/actors/hyrax/actors/generic_work_actor_spec.rb]: Later versions of rubocop-rspec suggests having blank lines after the subject and final let block.

File: spec/actors/hyrax/actors/generic_work_actor_spec.rb (hunk 1 above)
Comment (atz) [inline spec/actors/hyrax/actors/generic_work_actor_spec.rb]: @jcoyne I don't know if it is the jet lag or what, but this line is **added** by me, not deleted.  We're in agreement here.

Comment (jcoyne) [inline spec/actors/hyrax/actors/generic_work_actor_spec.rb]: Sorry, must be the jet lag.

File: spec/actors/hyrax/actors/generic_work_actor_spec.rb (hunk 2 above)
Comment (atz) [inline spec/actors/hyrax/actors/generic_work_actor_spec.rb]: @jcoyne We can deal with that when we get to later versions of rubocop-rspec with one big autocorrect, if we care.  For this terse little `let`/`it` pair though, I like it together.

**Pull request merged**